app/vmui/packages/vmui/src/utils/query-string.ts

```diff
@@ -105,7 +106,8 @@ export const getQueryStringValue = (
 
 export const getQueryArray = (): string[] => {
   const queryLength = window.location.search.match(/g\d+.expr/gmi)?.length || 1;
-  return new Array(queryLength).fill(1).map((q, i) => {
-    return getQueryStringValue(`g${i}.expr`, "") as string;
-  });
+  // .slice(0, MAX_QUERY_FIELDS)
```

Comment:
Remove this comment part of the code, please

Reply:
Fixed